src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonRequiredAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// Specifies the property is required when deserializing.
+    /// </summary>
```

Comment:
I think this needs a remarks section, I would possibly just take and adapt existing remarks from `JsonPropertyInfo.IsRequired` where applicable.